src/client/components/timePeriodPicker/TimePeriodPicker.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import RaisedButton from 'material-ui/RaisedButton';
+import PropTypes from 'prop-types';
+
+import './timePeriodPicker.css';
+import {
+  CHART_BUTTON_DEFAULT_LABEL_COLOR,
+  CHART_BUTTON_SELECTED_LABEL_COLOR,
+  TIME_PERIOD_FIVE_DAY,
+  TIME_PERIOD_FIVE_YEAR,
+  TIME_PERIOD_MAX,
+  TIME_PERIOD_ONE_DAY,
+  TIME_PERIOD_ONE_MONTH,
+  TIME_PERIOD_ONE_YEAR,
+  TIME_PERIOD_THREE_MONTH
+} from '../../../constants';
+
+const TimePeriodPicker = ({
+  chartTimePeriod,
+  updateChartTimePeriod
+}) => {
+  const handleClick = timePeriod => {
+    updateChartTimePeriod(timePeriod);
+  };
+  const setLabelColor = buttonLabel => {
+    if (chartTimePeriod === buttonLabel) {
+      return CHART_BUTTON_SELECTED_LABEL_COLOR;
+    }
+    return CHART_BUTTON_DEFAULT_LABEL_COLOR;
+  };
+  return (
+    <div className='timePeriodPicker'>
```

Comment:
Should generate all the RaisedButtons by looping over an array of the time periods, not typing them all out. DRY!!!!!